scripts/changelog.ts

```diff
@@ -290,8 +290,9 @@ if (typeof config.to === 'undefined') {
       break;
     case 'myaccount':
     case 'my-account':
-    case '@spartacus/my-account':
     case '@spartacus/myaccount':
+    case '@spartacus/my-account':
```

Comment:
What will be the name of the library in the end of the day? `@spartacus/myaccount` or `@spartacus/my-account?
@tobi-or-not-tobi which do you prefer?

Reply:
I vote for `my-account`.

Reply:
OK, we keep it as-is